lib/logstash/outputs/kusto.rb

```diff
@@ -119,10 +119,19 @@ class LogStash::Outputs::Kusto < LogStash::Outputs::Base
   # Check Proxy URL can be over http or https. Dowe need it this way or ignore this & remove this
   config :proxy_protocol, validate: :string, required: false , default: 'http'
 
+  # Maximum size (number of events) of the buffer before it gets flushed, defaults to 1000
+  config :max_size, validate: :number, default: 1000
+
+  # Maximum interval (in seconds) before the buffer gets flushed, defaults to 60
+  config :max_interval, validate: :number, default: 60
+
   default :codec, 'json_lines'
 
   def register
     require 'fileutils' # For mkdir_p
```

Comment:
How does it all add up ? It is just a buffer that is added.

Currently there is events written to file and then these files are flushed. With the new approach these files should go away

Right now the code changes are

a) Buffer that is taken from LA plugin
b) We write the events to the buffer and flush. How is this message encoded ?
c) can we remove the file part ? or the file buffers based on size on file ?
d) Where are the Tests ?
e) Time based flushes ?